conftest.py

```diff
@@ -31,7 +29,9 @@ def scan_manager(mocker):
 @pytest.fixture(scope="session")
 def django_client(live_server):
     """HTTP client which connects to django live server."""
-    client = BaseUrlClient(
+    from compat.requests import Session
```

Comment:
Why local import?

Reply:
the toplevel conftest.py has a policy of not importing quipucords code (unless it is under `tests` namespace). That's to avoid loading pesking global variables or messing with any django configuration that should happen later.

This also makes mocking easier.

Reply:
`api`, `quipucords` and `scanner` are the dangerous modules, tho.
Importing this particular one wouldn't break anything. at least not today.

Reply:
Fair enough! Thanks for the explanation. 🙂